packages/flutter_driver/lib/src/driver/web_driver.dart

```diff
@@ -182,7 +182,9 @@ class WebFlutterDriver extends FlutterDriver {
 
   @override
   Future<List<int>> screenshot({ScreenshotFormat format = ScreenshotFormat.png}) async {
-    assert(format == ScreenshotFormat.png, 'Web Driver only supports PNG screenshot format');
+    if (format != ScreenshotFormat.png) {
+      throw ArgumentError('Web Driver only supports PNG screenshot format');
```

Comment:
Use `ArgumentError.value` so the user sees what format was providwed.

Reply:
Cool